### DISCUSSION TITLE
Help needed

### DISCUSSION THREAD

**Discussion by Jay1530** [beetbox/beets#5155] [Q&A]
hi, 

First off I am a complete noob at coding and I'm trying my best to configure my Beets using the doc, but I have run into some problems. The first one is the gstreamer plugin. When I type ''pip install gstreamer'' I get this error:

ERROR: Could not find a version that satisfies the requirement gstreamer (from versions: none)
ERROR: No matching distribution found for gstreamer

I have managed to install ''gstreamer-player'', but I am not sure if this can be used for acoustic fingerprint.

The second problem is in my yaml file, more precisely in line 10 and 11 column 1 :

configuration error: C:\Users\jerem\AppData\Roaming\beets\config.yaml could not be read: while scanning a simple key
  in "C:\Users\jerem\AppData\Roaming\beets\config.yaml", line 10, column 1
could not find expected ':'
  in "C:\Users\jerem\AppData\Roaming\beets\config.yaml", line 11, column 1

Here is my code so far : 
`directory: ~/music/Music After Beet
library: ~/music/Music For Beet/newlibrary.blb
import:
    move: yes
threaded: yes
paths:
    default: $albumartist/$album%aunique{}/$track - $title ($bitrate)
    singleton: Non-Album/$artist/$title
    comp: Compilations/$albumartist/$album%aunique{}/$track - $title ($bitrate)
languages:en jp
plugins : fetchart lyrics discogs pyacoustid gstreamer`

I am on windows 10, if any other info is needed please ask!

**Comment by Jay1530** [comment]
here is my yaml file but in the right form :

directory: ~/music/Music After Beet
library: ~/music/Music For Beet/newlibrary.blb
import:
    move: yes
threaded: yes
paths:
    default: $albumartist/$album%aunique{}/$track - $title ($bitrate)
    singleton: Non-Album/$artist/$title
    comp: Compilations/$albumartist/$album%aunique{}/$track - $title ($bitrate)
languages:en jp
plugins : fetchart lyrics discogs pyacoustid gstreamer

**Reply by Jay1530** [reply]
It seems there was a missing space in the ''language'' line

**Comment by Jay1530** [comment]
Now I get this, even after rebooting : 

** error loading plugin pyacoustid:
Traceback (most recent call last):
  File "C:\Users\jerem\AppData\Local\Programs\Python\Python312\Lib\site-packages\beets\plugins.py", line 268, in load_plugins
    namespace = __import__(modname, None, None)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'beetsplug.pyacoustid'

** error loading plugin gstreamer:
Traceback (most recent call last):
  File "C:\Users\jerem\AppData\Local\Programs\Python\Python312\Lib\site-packages\beets\plugins.py", line 268, in load_plugins
    namespace = __import__(modname, None, None)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'beetsplug.gstreamer'

**Comment by arsaboo** [comment]
I suggest you add one plugin at a time to avoid multiple points of failure. Note that the plugin names are chroma (pyacoustid is the dependency and not the name of the plugin)and replaygain (gstreamer is a backend and not the plugin).

**Reply by Jay1530** [reply]
Thank you I am not familiar with these terms, like the naming of my directory and library doesn't really reflect what they actually are lol. I managed to install chroma, but the same error with replaygain : 

 C:\Users\jerem>pip install beets replaygain
Requirement already satisfied: beets in c:\users\jerem\appdata\local\programs\python\python312\lib\site-packages (1.6.0)
ERROR: Could not find a version that satisfies the requirement replaygain (from versions: none)
ERROR: No matching distribution found for replaygain

**Reply by arsaboo** [reply]
Please take a look at the documents for each plugin and follow the steps listed.

Chroma - https://beets.readthedocs.io/en/latest/plugins/chroma.html
replaygain - https://beets.readthedocs.io/en/latest/plugins/replaygain.html

Again, do this for one plugin at a time.

**Reply by Jay1530** [reply]
Now there's only one thing not working and it's the fpcalc:

chroma: fingerprint generation failed: fpcalc not found

I have put the fpcalc.exe files (even tried the whole folder and renaming it to fpcalc) in the Program files like the doc said :

Installing the Binary Command-Line Tool
The simplest way to get up and running, especially on Windows, is to [download](https://acoustid.org/chromaprint) the appropriate Chromaprint binary package and place the fpcalc (or fpcalc.exe) on your shell search path. On Windows, this means something like C:\\Program Files. On OS X or Linux, put the executable somewhere like /usr/local/bin.
